myia/opt/__init__.py

```diff
@@ -1,6 +1,6 @@
 """Optimization submodule."""
 
-from .clean import simplify_types, type_to_tag  # noqa
+from .clean import _strmap_tag, simplify_types, str_to_tag, type_to_tag  # noqa
```

Comment:
Don't expose private stuff.
```suggestion
from .clean import simplify_types, str_to_tag, type_to_tag  # noqa
```